commands/sense/lib/build.js

```diff
@@ -69,6 +69,21 @@ async function build(argv) {
     }
   };
 
+  const copySource = () => {
+    try {
```

Comment:
I suggest you skip the try/catch blocks here... making a `console.error(e)` will print the error message but the execution will continue. But if the copy commands are not going through, there are no point in continuing the way I see it.

Reply:
Updated!